packages/open-next/src/http/openNextResponse.ts

```diff
@@ -306,4 +306,23 @@ export class OpenNextNodeResponse extends Transform implements ServerResponse {
     }
     callback();
   }
+
+  get sent() {
+    return this.finished || this.headersSent;
+  }
+
+  getHeaderValues(name: string): string[] | undefined {
+    const values = this.getHeader(name);
+
+    if (values === undefined) return undefined;
```

Comment:
Would it be possible if the header was somehow set to `null`?

Reply:
It's mostly copied from next code directly, i assume it must be fine https://github.com/vercel/next.js/blob/6474cf0fe3046c148aceb7f6879c959d778457b0/packages/next/src/server/base-http/node.ts#L124-L132